### PR TITLE
[CR] Add default player sex option

### PR DESCRIPTION
**Pull request by jzambo**

### CODE REVIEW

File: src/newcharacter.cpp [hunk 1]
```diff
@@ -235,7 +235,14 @@ void player::randomize( const bool random_scenario, points_left &points )
     // Reset everything to the defaults to have a clean state.
     *this = player();
 
-    g->u.male = (rng(1, 100) > 50);
+    if (get_option<std::string>( "DEF_CHAR_SEX" ) == "male" ) {
+      g->u.male = true;
+    } else if (get_option<std::string>( "DEF_CHAR_SEX" ) == "female" ) {
+      g->u.male = false;
+    } else {
+      g->u.male = (rng(1, 100) > 50);
```
Comment (BrettDong) [inline src/newcharacter.cpp]: `one_in(2)` is clearer.

**Comment by Mecares** [comment]
Personally i would prefer if the game would just use the sex of the previous character as default when creating a new character over having an entry in the  options just for this.

**Comment by Coolthulhu** [comment]
Doesn't seem to work. I set the setting to "female" (it's the second one and so should catch more bugs), but the chargen defaulted to male anyway.

**Comment by jzambo** [comment]
Ah, thanks for catching this. It must be because I "Play Now!" with random characters, so I must have missed some other place to update. Any hints before I start digging?

**Comment by Leland** [comment]
Closing, seems stalled.

If you're looking for code help, give the forums a shot: https://discourse.cataclysmdda.org/

**Pull request closed without merging**